candidate-standard/abstract_tests/covjson/ATS_content.adoc

```diff
@@ -6,6 +6,5 @@
 ^|Requirement |<<req_covjson_content,/req/covjson/content>>
 ^|Test Method |. Validate that the document is a CoverageJSON document.
 . Validate the document against the schema using an JSON Schema validator.
-. Validate the document against the schema using an GeoJSON Schema validator.
 . Validate the document against the schema using an CoverageJSON Schema validator.
```

Comment:
Should this be just "against the CoverageJSON schema"?

Reply:
Yes. CoverageJSON may or may not be valid GeoJSON.